### DISCUSSION TITLE
Entire docker folder moved to another server -> page "Not found"

### DISCUSSION THREAD

**Discussion by maiducan** [Budibase/budibase#8574] [Help]
Hello, I want to duplicate my budibase app to another server.

I copied entire docker folder /var/lib/docker to new machine. Altogether with docker-compose.yml and .env

Then i started docker there using budi hosting --start based on the settings from yl, all containers loaded but i got blank page "Not found" when I access the web. 

However Minio web is working for example .. So it must be some problem in app container

Any tips? Thanks guys, budibase is the best!

**Comment by melohagan** [comment]
Hey @maiducan 

You may want to have a read through this: https://docs.budibase.com/docs/backups

**Reply by maiducan** [reply]
Thanks, ill try that one.

**Reply by maiducan** [reply]
<img width="1310" alt="image" src="https://user-images.githubusercontent.com/8984651/200766494-ca3e2ea2-2aee-43dd-845f-0e067e81eaaf.png">

I tried the backup feature and it failed with memory heap javascript, any tips?
I have 6GB RAM on my server, and it uses max 50% of it

**Reply by melohagan** [reply]
Ah that heap error is possibly caused by a limit within your NodeJS config. This article may help: https://www.makeuseof.com/javascript-heap-out-of-memory-error-fix/

**Comment by maiducan** [comment]
Is there any way i can get the previous process working?

Budi CLI is crashing for me due to the size of my volumes ... :/

**Reply by shogunpurple** [reply]
Can you not just export/import your app to the other server?

**Reply by shogunpurple** [reply]
https://docs.budibase.com/docs/export-and-import-apps

**Reply by maiducan** [reply]
The export app crashes after several minutes as well :/ (502, Houston we have problem) 
Nginx timeouts are set high

**Reply by shogunpurple** [reply]
How big is your app? Have you tried exporting without rows?

I’m just trying to understand if this is actually a Budibase issue or an issue with your self hosted setup.

**Reply by maiducan** [reply]
The same result with excluded rows :/

I am using BB DB because of the attachments
The biggest BB table has 5K rows (each has photo attachments)

I should have used postgress and S3 from the beginning for all tables, BB db started to be very slow after certain amount of data, but i had to keep it due to those attachments ..

**Reply by shogunpurple** [reply]
I'm having a very hard time recreating this - I can't seem to do it even with large apps on my own self hosted instance and in our cloud environments. 5000 isn't actually a lot of rows either, so shouldn't be taking minutes. 

Can you export apps at all? So if you even create a fairly simple app in your instance with some basic screens, can you export that one?

**Reply by maiducan** [reply]
I have created a new small app in the same space and it exported immediately without problem.

5K isnt much, but maybe a lot of data inside of each - when i exported just the table to .csv, it was a file with 34K rows.

From the experience, the time of the app export grew linearly last few months with the size of that table. I was able to export it 2 months ago, but now i guess it reached some threshold and it crashes on 502.

**Reply by shogunpurple** [reply]
Hm there's a chance you are hitting the NGINX problems due to the body size.

https://github.com/Budibase/budibase/discussions/4912
and
https://github.com/Budibase/budibase/discussions/9121

Contain information on how to do this.

**Reply by maiducan** [reply]
- large entity did not help, i have already 200M
- the second one i dont know how to do

**Comment by maiducan** [comment]
I think there is just some little tweak how I can make the first process work .. 

Because sometimes on my production, when i restart docker, it land to that "Not found" page as well, but after a second it automatically starts the platform and it works .. So maybe if you can tell me what happens after that "Not found" page, it could help me to make it work.

**Comment by maiducan** [comment]
@shogunpurple Hello, i think this is crucial issue, a lot of users are gonna have the same problem in near future, when their apps will grow .. Can you please fix the export app at least, so i have have a backup. Now its very dangerous for me, I use budibase for my SaaS and its in the point where i dont have 2 months app backup .. If anything happens, my whole business is gonna die .. :/
